feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/following/FollowingFragment.java

```diff
@@ -1,33 +1,176 @@
 package com.example.feelslikemonday.ui.following;
 
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
 import android.os.Bundle;
+import android.view.Gravity;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-
+import android.widget.Toast;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.Observer;
 import androidx.lifecycle.ViewModelProviders;
-
+import com.baoyz.swipemenulistview.SwipeMenuListView;
+import com.example.feelslikemonday.DAO.FollowPermissionCallback;
+import com.example.feelslikemonday.DAO.FollowPermissionDAO;
+import com.example.feelslikemonday.DAO.UserCallback;
+import com.example.feelslikemonday.DAO.UserDAO;
+import com.example.feelslikemonday.DAO.VoidCallback;
 import com.example.feelslikemonday.R;
+import com.example.feelslikemonday.model.FollowPermission;
+import com.example.feelslikemonday.model.MoodEvent;
+import com.example.feelslikemonday.model.User;
+import com.example.feelslikemonday.model.followeeMoodEvent;
+import com.example.feelslikemonday.service.SortObjectDateTime;
+import com.example.feelslikemonday.ui.home.DisplayCurrentMood;
+import com.example.feelslikemonday.ui.login.SignupActivity;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import com.baoyz.swipemenulistview.SwipeMenu;
+import com.baoyz.swipemenulistview.SwipeMenuCreator;
+import com.baoyz.swipemenulistview.SwipeMenuItem;
+
+/**
+ * Responsible for listing all followees' moods and related information
+ */
 
-/*Responsible for listing all followers' moods and related information */
 public class FollowingFragment extends Fragment {
 
     private FollowingViewModel followingViewModel;
+    private SwipeMenuListView userList;
+    private FolloweeBookAdapter adapter;
+    private List<String> followeeUsernames;
+    private static FollowPermissionDAO DAO;
+    private SharedPreferences pref;
+    private String myUserID;
+    private List<MoodEvent> followeeUserMoodList;
+    private ArrayList<followeeMoodEvent> myfolloweeList = new ArrayList<>();
+    private int xIteration;

```

Comment:
xInteration does not need to be an attribute since it's only used in a function

Reply:
ok got rid of it as an attribute